node/generate_release.js

```diff
@@ -0,0 +1,32 @@
+const { readFileSync } = require('fs');
+const { join } = require('path');
```

Comment:
I prefer CJS.  Does it matter?

Reply:
I figured we had a bit of a pattern going with all our other semi-recent to a yr+ old scripts written in `.mjs` format using `node:` style imports for the sake of producing new code in the modern format rather than new code in an old format. 

It is a nice-to-have to stick to the "new" (2017) rather than introduce more old, but not a blocker.